modules/user_accounts/templates/form_my_preferences.tpl

```diff
@@ -1,12 +1,14 @@
 <br />
 <form method="post" name="edit_user" id="edit_user">
     <h3>Password Rules</h3>
-    <ul>
-        <li>The password must be at least 8 characters long</li>
-        <li>The password must contain at least 1 letter, 1 number and 1 character from   !@#$%^&amp;*()</li>
-        <li>The password and the user name must not be the same</li>
-        <li>The password and the email address must not be the same</li>
-    </ul>
+      <ul>
+        <li>The password must be at least 8 characters long.</li>
+        <li>The password and the user name must not be the same.</li>
+        <li>The password and the email address must not be the same.</li>
+      </ul>
+        <p><b>Please choose a unique password.</b></p>
+        <p>We suggest using a password manager to generate one for you.</p>
+      </ul>
```

Comment:
Your tags aren't matched up.. you're closing 2 `<ul>` tags but only opening one. Styling should also be done with CSS instead of presentational tags (ie. `<b>` shouldn't be used..)

Reply:
Alrighty, I just copied from the login module so that should maybe be fixed at some point